br/pkg/lightning/backend/local/localhelper_test.go

```diff
@@ -260,8 +259,8 @@ func (c *testClient) ScanRegions(ctx context.Context, key, endKey []byte, limit
 	regions := make([]*restore.RegionInfo, 0, len(infos))
 	for _, info := range infos {
 		regions = append(regions, &restore.RegionInfo{
-			Region: info.GetMeta(),
-			Leader: info.GetLeader(),
+			Region: info.Meta,
+			Leader: info.Leader,
```

Comment:
Why change these?

Reply:
Because I did not add methods for the type (https://github.com/pingcap/tidb/pull/31617/files/0840be26a3ed2a0d9c73d5de81c6b617ecc09ae5#diff-9854a0ff1dab17c12d29b6a5c5cf19b859ab8f4651d1d1e1b8b2ece3842455eaR25)